static/jep/js/main.js

```diff
@@ -1,4 +1,8 @@
 
 $(function () {
   $('[data-toggle="tooltip"]').tooltip()
-})
\ No newline at end of file
+})
+
+$('#formset').on('formAdded', function(event) {
```

Comment:
@felixrindt this event listener is never executed, but the code inside would do the right thing. could you please have a look?

Reply:
/me gets their magic javascript gloves